test/python/circuit/test_extensions_standard.py

```diff
@@ -851,6 +850,55 @@ def test_z_reg_inv(self):
         self.assertEqual(instruction_set[1].qubits, (self.qr[1],))
         self.assertEqual(instruction_set[2].operation.params, [])
 
+    def test_global_phase(self):
+        qc = self.circuit
+        qc.append(GlobalPhaseGate(0.1), [])
+        self.assertEqual(self.circuit[0].operation.name, "global_phase")
+        self.assertEqual(self.circuit[0].operation.params, [0.1])
+        self.assertEqual(self.circuit[0].qubits, ())
+
+    def test_global_phase_inv(self):
+        instruction_set = self.circuit.append(GlobalPhaseGate(0.1), []).inverse()
+        self.assertEqual(len(instruction_set), 1)
+        self.assertEqual(instruction_set[0].operation.params, [-0.1])
+
+    def test_global_phase_matrix(self):  # , theta: float, expected: np.ndarray
+        """Test global_phase matrix."""
+        theta = 0.1
+        np.testing.assert_allclose(
+            np.array(GlobalPhaseGate(theta)),
+            np.array([[np.exp(1j * theta)]], dtype=complex),
+            atol=1e-7,
+        )
+
+    def test_global_phase_consistency(self):
+        from qiskit.quantum_info import Operator
+
```

Comment:
Please could you move this (and in the other test) import up to the top of the file?  We only use scoped imports like this where it's necessary to prevent cyclic imports, but in the test suite, that's never an issue.

Reply:
Thank you @jakelishman. I have made the changes. Thank you so much for the detailed explanations.